projects/distributed-tracing/src/shared/dashboard/data/graphql/table/table-data-source.model.ts

```diff
@@ -19,9 +26,9 @@ export abstract class TableDataSourceModel extends GraphQlDataSourceModel<TableD
 
   public getData(): Observable<TableDataSource<TableRow, SpecificationBackedTableColumnDef>> {
     return observableOf({
-      getData: request =>
-        this.query(filters => this.buildGraphQlRequest(filters, request)).pipe(
-          map(response => this.buildTableResponse(response, request))
+      getData: (request, mode) =>
```

Comment:
So this is where it flushes out, but we have active mode available in the renderer that could pass this down - which we're already doing for getting columns. Or we could do something else like switch to a different data source for flattened data, set a variable, etc.